apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/Admin.cs

```diff
@@ -53,5 +53,96 @@ public AdminTests(TestDatabaseFixture testFixture, ITestOutputHelper outputHelpe
                 await dropCommand.ExecuteNonQueryAsync().ConfigureAwait(false);
             }
         }
+
+        [Fact]
+        public async Task CreateWithExtrasDrop()
+        {
+            string dbName = "t_" + Guid.NewGuid().ToString("N").Substring(0, 28);
+
+            using (var connection = new SpannerConnection(_testFixture.NoDbConnectionString))
+            {
+                const string tableCreate = @"CREATE TABLE TX (
```

Comment:
So the "extra" statements are implicitly referring to the newly created database, yes? (Makes sense.)

Reply:
yes.  this feature is supported natively by the createdb admin endpoint.